packages/lit-dev-content/site/tutorials/content/working-with-lists/00.md

```diff
@@ -0,0 +1,24 @@
+This tutorial will be exploring different ways of rendering a list of items in
+Lit!
+
+Lit makes it easy to render any lists of items. It has built-it support for
+iterables of any supported values. This, along with Lit's ability to render
+nested templates and helpful built-in directives, allow for rendering any
+variety of lists.
+
+## What you'll learn
+- Utilizing the `map` directive
+- Using array prototype methods
+- Building static list and using the `range` directive
+- Imperatively building an array of items to render
+- Utilizing the `repeat` directive
```

Comment:
"Utilizing" is repeated here. I wonder if we want to add more description of what readers will do with each thing, like:

```
 - Rendering lists with the map() directive
 - Transforming arrays into templates with array.map()
 - Generating lists with the range() directive
 - Imperatively building arrays of items
 - Diffing lists of stateful elements with the repeat() directive
 - Using event listeners in a list

Reply:
@arthurevans do you have thoughts on whether we should add the `()` after the directive names? Scanning through https://lit.dev/docs/templates/directives/ has examples of both with and without the parens.

Reply:
That's a good question. Originally I thought we were going to go with parens after both method and directive names. The current state is inconsistent for both, unfortunately. I lean towards using the parens everywhere. But at any event, make sure they're consistent in your tutorial. I'll put this on the agenda for a future docs meeting.

Reply:
I'm personally against parens even for methods because the arguments may be important and it'd be kinda silly to include the method signature when simply referring to it

Reply:
Yeah, using empty parens is kind of convention for marking, "this is a function/method". It's recommended by the Google developer docs style guide, which we approximately follow:

https://developers.google.com/style/code-in-text#methods

Back in the polymer days, I bucked this trend because I'd worked on Java interfaces in the past and the JavaDoc rule is that you don't include empty parens--because of the way Java function overloading works, `foo()` may literally be a different function than `foo(bar)`. 

At present, it seems like like the JS world is divided on the value of the parens. React docs appear to use them: https://reactjs.org/docs/rendering-elements.html Vue docs don't. Svelte seems to use them for platform APIs like `event.preventDefault()`, but not for Svelte's own APIs, which seems odd. (Also, they prefix their own method names with a `.` to I guess show it's a member of an object, which also looks odd to me, so let's not do that.)

I don't have a super strong feeling about this one other than a general desire to fall in line with the developer docs style guide. But whatever we do, we should do it consistently.

Reply:
I can see arguments for both but lean towards @e111077's reasoning. I currently have consistently no parens within this tutorial and will keep it that way for now then.

Semi related to this, is do we have patterns for adding `.` in front of prototype methods like when referring to `.map`, `.filter` for arrays? For Step 3 of this tutorial, I've added the `.` if anything at least to make it stand out from the `map` directive.

Reply:
We've never used that pattern that I'm aware of. It looks like it's just the one place you mention them, so I'd suggest saying something like: "the `filter` or `map` array methods". It seems to me that `.map` might be reasonably clear if the reader was aware you were trying to make the distinction between the map array method and the map directive. 

But the dot is easy to miss and not necessarily transparent to a more novice programmer. I'd suggest erring on the side of being a bit more explicit.

Reply:
Ooh I did not see that comment about it being in the Google developer docs style guide. I'd be happy to lean on that for this tutorial then. Thanks for the link! I'll also update the array method portion.